cli/commands/usage.go

```diff
@@ -127,7 +140,7 @@ Ignored if "--stateboard-only" is specified
 `
 
 	stateboardOnlyUsage = `Manage only application stateboard
-If specified, "INSTANCE_NAME..." are ignored.
```

Comment:
Why did you do it?

Reply:
Because I decided to remove trailing dots.

Reply:
Is it only one place?

Could you please move such refactoring to separate patch if there are other places?

Reply:
I've done it in the previous patch, but missed this place.